test/src/test/java/hudson/model/ParametersTest.java

```diff
@@ -75,7 +76,7 @@ public void parameterTypes() throws Exception {
         Object o = DomNodeUtil.selectSingleNode(element, ".//input[@name='value']");
         System.out.println(o);
         HtmlCheckBoxInput booleanParameterInput = (HtmlCheckBoxInput) o;
-        assertEquals(true, booleanParameterInput.isChecked());
+        assertTrue(booleanParameterInput.isChecked());
```

Comment:
```suggestion
        assertThat(booleanParameterInput.isChecked(), is(true));
```
You can move this to hamcrest too like previous files.

Reply:
I really like the simplicity of assertTrue